game/player.py

```diff
@@ -15,7 +15,7 @@ class Player:
     def __init__(self):
         """Fills the player's team with a random Pokemon from the PokeAPI"""
         if len(self.team) < 1:
-            self.team.extend(asyncio.run(pokeapi.get_random_pokemon_from_api()))
+            self.team.extend(asyncio.run(pokeapi.get_random_pokemon_from_api(6)))
```

Comment:
This shouldn't be here

Reply:
Yes it should

Reply:
Changed to  self.team.extend(asyncio.run(pokeapi.get_random_pokemon_from_api(1)))

Reply:
Changed to  self.team.extend(asyncio.run(pokeapi.get_random_pokemon_from_api(1))) to remove redundancy